requirements.txt

```diff
@@ -49,7 +49,8 @@ PyYAML; python_version >= '2.7'
 pycodestyle; python_version < '2.7'
 flake8; python_version >= '2.7'
 
-GC3Pie
+# 2.6.7 uses invalid Python 2 syntax
+GC3Pie!=2.6.7
```

Comment:
We can make this a bit stricter, and keep testing with GC3Pie (current latest) when testing with Python 3 (for what it's worth...)

```
GC3Pie!=2.6.7; python_version <= '2.7'
GC3Pie; python_version > '2.7'